vendor/extensions/reports/app/controllers/refinery/reports/reports_controller.rb

```diff
@@ -6,17 +6,22 @@ class ReportsController < ::ApplicationController
       before_action :find_page
 
       def index
-        # you can use meta fields from your model instead (e.g. browser_title)
-        # by swapping @page for @report in the line below:
-        present(@page)
+        reports_json = @reports.map { |report| ReportSerializer.new(report, { :include => [:image] }).serializable_hash }
+        
+        respond_to do |f|
+          f.html { present(@page) }
+          f.json { render json: reports_json }
+        end
       end
-
+      
```

Comment:
As you are going through your files/changes could you delete this white space? I thought we enabled removing it automatically in VSCode, but if we did not do that then I found the setting at https://stackoverflow.com/a/30884298/2149359 which could help fix these automatically.